src/os/ui/featureedit.js

```diff
@@ -317,6 +319,12 @@ os.ui.FeatureEditCtrl = function($scope, $element, $timeout) {
     os.math.Units.METERS
   ];
 
+  /**
+   * Units for altitude
+   * @type {string}
+   */
+  this['altOptions'] = goog.object.getKeys(os.math.Units);
```

Comment:
Use the values of `os.math.Units` instead. The keys will be minified by the Closure compiler, resulting in confusing options for the user.

For an example of another UI with altitude units, see `src/os/ui/wiz/geometrystep.js`.